dracut/30ignition/coreos-remove-initramfs-network.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# -*- mode: shell-script; indent-tabs-mode: nil; sh-basic-offset: 4; -*-
+# ex: ts=8 sw=4 sts=4 et filetype=sh
+
+# Clean up the interfaces set up in the initramfs
+for f in /sys/class/net/*; do
```

Comment:
Hmm, this should probably be strengthened to handle the case where there are no network interfaces at all.

Reply:
I believe for this particular use case, if there is no network interfaces at all, NM will correctly bring up interfaces for the real root? Or is there some specific scenarios you are thinking of.

I meant for this PR to handle only network interfaces right now, since it is the first step (and unblocks static IP generation). I agree that the functionality needs to be strengthened a lot more (maybe in followup PRs?)

Reply:
> Or is there some specific scenarios you are thinking of.

If a directory is empty, then `for f in /my/dir/*; do` will still iterate once with that string verbatim as `f`. Though hmm, I guess we can always count on at least `lo` to be there.